securedrop/tests/test_source.py

```diff
@@ -522,6 +522,7 @@ def test_metadata_route(source_app):
         assert resp.headers.get('Content-Type') == 'application/json'
         assert json.loads(resp.data.decode('utf-8')).get('sd_version') \
             == version.__version__
+        assert resp.get('server_os') == 16.04
```

Comment:
since we want these tests to pass on both trusty/14.04 and xenial/16.04 (we run tests in both environments), can you mock out `platform.linux_distribution`? Check out [this commit](https://github.com/freedomofpress/securedrop/pull/4094/commits/6b579b8e0baa6b6c48c5b6187574f783d42276e9) for a example of how to do that

Reply:
Yes, will try my best

Reply:
![screenshot from 2019-02-04 04-41-58](https://user-images.githubusercontent.com/32111606/52184324-d4cc9380-2837-11e9-8c5e-dc0c10a5ca1a.png)
@redshiftzero Why is it failing ?

Reply:
> @redshiftzero Why is it failing ? Is it the because of mock .

Did you test the test case locally by running all of the tests? 

Check https://circleci.com/gh/freedomofpress/securedrop/21953 and see the error message in `red`. If you still have questions, ask in gitter. Being able to read error messages will help in long run. Feel free to ping me on IRC if you want.